infrastructure/nomad-configuration/nomad-client-instance-user-data.tpl.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
+
+# This is a template for the instance-user-data.sh script for the nomad client.
+# For more information on instance-user-data.sh scripts, see:
+# https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/user-data.html
+
+# This script will be formatted by Nomad, which will read files from
+# the project into terraform variables, and then template them into
+# the following Here Documents. These will then be written out to files
+# so that they can be used. A more ideal solution than this would be if
+# we could just give AWS a list of files to put onto the instance for us,
+# but they only give us this one script to do it with. Nomad has file
+# provisioners which will put files onto the instance after it starts up,
+# but those run after this script runs.
+
+
+# Change to home directory of the default user
+cd /home/ubuntu
+
+
+# Output the files we need to start up Nomad and register jobs:
+# (Note that the lines starting with "$" are where
+#  Terraform will template in the contents of those files.)
+
+# Create the script to install Nomad.
+cat <<"EOF" > install_nomad.sh
+${install_nomad_script}
+EOF
+
+# Create the Nomad Client configuration.
+cat <<"EOF" > client.hcl
+${nomad_client_config}
+EOF
+
+
+# Create a directory for docker to use as a volume.
```

Comment:
Why is this /tmp? Does the volume need to persist?

Reply:
Hmm, good point. Long term it won't matter because we'll be using EFS, but I may as well just create this directory in `/home/ubuntu` anyway.